pkg/bufferdecoder/eventsreader.go

```diff
@@ -367,9 +351,10 @@ func readStringFromBuff(ebpfMsgDecoder *EbpfDecoder) (string, error) {
 	return string(res), nil
 }
 
-// readStringVarFromBuff reads a null-terminated string from `buff`
-// max length can be passed as `max` to optimize memory allocation, otherwise pass 0
-func readStringVarFromBuff(decoder *EbpfDecoder, max int) (string, error) {
+// readVarStringFromBuffer reads a null-terminated string from the ebpf buffer where the size is not
+// known. The helper will read from the buffer char-by-char until it hits the null terminator
+// or the given max length. The cursor will then skip to the point in the buffer after the max length.
+func readVarStringFromBuffer(decoder *EbpfDecoder, max int) (string, error) {
```

Comment:
You already know that this is being touched in https://github.com/aquasecurity/tracee/pull/4634/files#diff-d9cde7aaf9961101ae53f694c6932d5943eceb87ed114291d51e88ec2d535969R376. I'm just referencing it for both of us (more for myself) to remember later.